index.html

```diff
@@ -34,7 +35,7 @@
         go home
       </a>
       <div>
-        <a href="resume.html">
+        <a href="/resume">
```

Comment:
looks better on-screen and also bc i made the canonical the one without the `.html`... if it doesn't work then revert lol